### DISCUSSION TITLE
想知道下摘录卡片要如何制作？

### DISCUSSION THREAD

**Discussion by Ohcecho** [oflg/fishing#41] [General]
我知道问答卡片需要加上标签"?"，但不知道摘录卡片要如何制作，是直接使用标签"."吗？但这样复习的时候好像就不能跟问答卡片一起复习了。我尝试过在卡片同时添加标签"?""."，但这样卡片的选项就会在”忘记“、”记得“、”掌握“前面出现”学习“，不知道这样是否正常。另外我想知道摘录卡片是否是每隔24小时出现一次，如果是我使用了上述的第二种方式，在我复习后，这张卡片会根据我的复习情况出现不同的复习时间安排。所以我想知道正确制作摘录卡片的方式是什么。感谢解答。

**Comment by oflg** [comment]
出现”学习“是否正常：

- 出现`学习`和`丢弃`是正常的，这个`学习`用词并不是很准确，因为它的功能是“保留这个卡片并展示下一个卡片”，`丢弃`的功能是“不再间隔重复这个摘录卡片”。摘录卡片不会出现”忘记“、”记得“、”掌握“（这个只在问答卡片上有），会一直是`学习`和`丢弃`。

摘录卡片是否是每隔24小时出现一次：

- 不是。只有第一次是1天（可[自定义](https://tiddlymemo.org/manual/zh-Hans#%E5%A6%82%E4%BD%95%E6%8C%89%E7%BB%84%E5%88%AB%E5%AD%A6%E4%B9%A0%E6%9D%A1%E7%9B%AE%EF%BC%9F)），重复次数越多，间隔越长

正确制作摘录卡片的方式：

- 正确制作摘录卡片的方式是加`?`和`.`，来制作可以间隔重复的摘录卡片。
- 加`.`制作只使用一次的摘录卡片。

**Reply by Ohcecho** [reply]
我刚刚实验了一下，貌似是我在使用标签"."后，开始第一次复习的时候再加上标签"?"，会出现“学习”和“忘记”“记得”“掌握”同时出现的情况，非常感谢你的解答；另外想问下，有没有可以自定义快捷键的方式，比如说我按下某快捷键会出现一张全新的问答卡片（相当于创建一张新条目并加上标签“?”），这是否需要学习tw相关的编程语言？

**Reply by oflg** [reply]
有一个简单的方法是还用系统原来的快捷键，只用配置下图中两个选项中任一个（推荐给日志条目配上`?`标签，快捷键alt+j）

![image](https://github.com/oflg/fishing/assets/42492105/4ecbba5c-7d03-441d-afb1-f06ec7abc3fc)
